app/main.py

```diff
@@ -1,2 +1,76 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self,
+                 initial_capacity: int = 8,
+                 load_factor: float = 0.75
+                 ) -> None:
+        self.initial_capacity = initial_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.capacity = initial_capacity
+        self.table = [None] * self.capacity
+
+    def __setitem__(self,
+                    key: int | float | str | bool | tuple,
```

Comment:
There are more types that could be dict keys. Keys should be _**hashable** types_

Reply:
I'm hinting to `hashable` type annotation